sdk/monitor/azure-monitor-ingestion/swagger/README.md

```diff
@@ -2,13 +2,19 @@
 
 ## Code generation settings
 
+### Manual Modifications
+
+The following edits need to be made manually after code generation:
+- Rollback the edits to `module-info` file
+
 ```yaml
 java: true
-use: '@autorest/java@4.1.9'
```

Comment:
@srnagar  I'll pin it to the latest, but can you tell me why we should do this? Autorest versions are not auto-updated like other dependencies and I've seen that a lot of other SDKs are using outdated autorest versions because nobody is keeping an eye on them.